lib/src/clojure_lsp/feature/semantic_tokens.clj

```diff
@@ -150,22 +151,32 @@
              alias)
          (not namespace-from-prefix))
     (let [slash (+ (if alias 2 1) name-col (count (str (or alias ns))))
-          ns-pos (assoc element :name-end-col slash)
+          ns-pos (-> element
+                     (assoc :name-end-col slash)
+                     (assoc :name-col (+ (if alias 2 1) (:name-col element))))
           slash-pos (assoc element :name-col slash :name-end-col (inc slash))
           name-pos (assoc element :name-col (inc slash))]
       [(element->absolute-token ns-pos :type)
        (element->absolute-token slash-pos :event)
        (element->absolute-token name-pos :keyword)])
 
     :else
-    [(element->absolute-token element :keyword)]))
+    (as-> element $
+      (assoc $ :name-col (if (and ns
+                                  (not namespace-from-prefix))
+                           (+ name-col 2)
+                           (inc name-col)))
+      [(element->absolute-token $ :keyword)])))
 
 (defn ^:private elements->absolute-tokens
   [elements]
   (->> elements
        (sort-by (juxt :name-row :name-col))
        (keep
          (fn [{:keys [bucket] :as element}]
+           (when (= (:name-row element) 176)
+             (logger/info element))
+
```

Comment:
?
```suggestion
```

Reply:
I removed this statement but I didn't push it 😅